.github/workflows/infection.yml

```diff
@@ -22,17 +22,15 @@ jobs:
       - name: Use PHP ${{ env.latest_php }}
         uses: shivammathur/setup-php@v2
         with:
-          coverage: none
           php-version: ${{ env.latest_php }}
           extensions: apcu
+          ini-values: xdebug.mode=coverage
```

Comment:
Is this still needed with no debug?

Reply:
yes, we still need this, since this is not the default xdebug mode. i tried to do it inline on the phpunit command, but that didn't seem to have worked. besides, this looks cleaner to me.